app/views/curation_concerns/base/_form_representative.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="row">
```

Comment:
@jcoyne They are a copy from CurationConcerns. I removed a `<div class="col-md-12">` but they would probably render OK with that div.